topydo.conf

```diff
@@ -6,6 +6,8 @@ default_command             = ls
 ; archive_filename          = done.txt
 colors                      = 1
 identifiers                 = linenumber ; or: text
+console_height              = 25
```

Comment:
I'm not opposed to determining console size at run time, but we need to consider:
1. what happens when we run this on Windows? `stty` isn't available, by default, on Windows systems
2. what happens if we run somewhere that isn't a terminal, like even the test suite?

I'll keep looking to see if I can find a multiplatform solution.